docs/strategy-customization.md

```diff
@@ -791,6 +793,33 @@ The following lists some common patterns which should be avoided to prevent frus
 - don't use `dataframe['volume'].mean()`. This uses the full DataFrame for backtesting, including data from the future. Use `dataframe['volume'].rolling(<window>).mean()` instead
 - don't use `.resample('1h')`. This uses the left border of the interval, so moves data from an hour to the start of the hour. Use `.resample('1h', label='right')` instead.
 
+### Performance warning
+
+When executing a strategy, one can sometimes be greeted by the following in the logs
+
+> PerformanceWarning: DataFrame is highly fragmented.
+
+This is a warning from [`pandas`](https://github.com/pandas-dev/pandas) and as the warning continues to say:
+ use `pd.concat(axis=1)`. For example
+
+```python
+for i in range(100):
+   dataframe[i] = ta.indicator(dataframe, param=i)
+```
+
+should be rewritten to
+
+```python
+frames = [dataframe]
+for i in range(100):
+   frame = DataFrame()
+   frame[i] = ta.indicator(dataframe, param=i)
+   frames.append(frame)
+
+# Append column to existing dataframe
+merged_frame = pd.concat(frames, axis=1)
```

Comment:
unfortunately, this will only help in very specific cases - but not in the cases most people will encounter.

You can get this warning also when calculationg 30 different indicators (not sure where exactly the threshold lies) - but as that's then no longer a loop, your solution is not really applicable.

In my opinion / understanding of this problem, it's a warning that's pointless to how freqtrade uses the dataframe.

The only problem it causes is the output itself - performance of the dataframe does not suffer (it'll happen in backtesting / hyperopt - where only a subset of the available columns are used for further processing).


So to me, a proper fix to this will disable the warning in pandas - not try to document it away.